src/test/scala/com/linkedin/spark/datasources/tfrecord/TFRecordDeserializerTest.scala

```diff
@@ -1,69 +1,134 @@
 /**
- * Copyright 2016 The TensorFlow Authors. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+  * Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *       http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
 package com.linkedin.spark.datasources.tfrecord
 
 import com.google.protobuf.ByteString
 import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.catalyst.util.{ArrayData, GenericArrayData}
 import org.apache.spark.sql.types._
 import org.apache.spark.unsafe.types.UTF8String
-import org.scalatest.{Matchers, WordSpec}
 import org.tensorflow.example._
 import TestingUtils._
+import org.scalatest._
+import matchers.should._
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class TFRecordDeserializerTest extends AnyWordSpecLike with Matchers {
```

Comment:
New ScalaTest version.

I think my IDE reformatted things. Let me know if there are specific formatting options I should use and I can revert.